src/typeDefs/types.ts

```diff
@@ -202,14 +223,15 @@ export const types = gql`
     description: String!
     location: String
     isPublic: Boolean!
-    creator: User!
+    creator: User
+    createdAt: DateTime!
+    updatedAt: DateTime!
     members: [User]
-    admins(adminId: ID): [User]
+    admins(adminId: ID): [User!]!
     membershipRequests: [MembershipRequest]
     blockedUsers: [User]
     visibleInSearch: Boolean!
-    apiUrl: URL!
-    createdAt: DateTime
+    apiUrl: URL
```

Comment:
make this `apiUrl: URL!`

`URL` is a graphql scalar, it's just a normal string that is validated within the schema to be some kind of url